src/components/cc-global-header.vue

```diff
@@ -31,17 +31,9 @@
-                        </g>
-                      </use>
-                    </svg>
-                  </div>
+             <div class="navbar-brand column is-one-fifth">
+                <a class="main-logo" href="/" target="_blank">
+                    <img src="../../assets/cc_logo_black.png" class="w-full" alt="Creative Commons Logo">
```

Comment:
Similarly here, we may need to figure out how to link to the image in the compiled/published package.